tests/test_enki.py

```diff
@@ -0,0 +1,127 @@
+from nose.tools import (
+    assert_raises_regexp,
+    set_trace,
+    eq_,
+)
+import datetime
+import os
+import pkgutil
+from core.model import (
+    CirculationEvent,
+    Contributor,
+    DataSource,
+    LicensePool,
+    Resource,
+    Hyperlink,
+    Identifier,
+    Edition,
+    Timestamp,
+    Subject,
+    Measurement,
+    Work,
+)
+from . import DatabaseTest
+from api.circulation_exceptions import *
+from api.enki import (
+    EnkiAPI,
+    MockEnkiAPI,
+    EnkiBibliographicCoverageProvider,
+)
+from core.scripts import RunCollectionCoverageProviderScript
+from core.util.http import BadResponseException
+from core.testing import MockRequestsResponse
+
+class BaseEnkiTest(object):
+
+    base_path = os.path.split(__file__)[0]
+    resource_path = os.path.join(base_path, "files", "enki")
+
+    @classmethod
+    def get_data(cls, filename):
+        path = os.path.join(cls.resource_path, filename)
+        return open(path).read()
+
+
+class TestEnkiAPI(DatabaseTest, BaseEnkiTest):
+
+    def setup(self):
+        super(TestEnkiAPI, self).setup()
+        self.collection = self._collection(protocol=EnkiAPI.ENKI)
+        self._collection(protocol=EnkiAPI.ENKI)
```

Comment:
Oh you're right. You can still remove the second line though - you only need the first one that sets self.collection.